utils/fcm_utils_test.go

```diff
@@ -0,0 +1,107 @@
+package utils
```

Comment:
I had some trouble mocking the Messaging client so that's the only part that doesn't have a unit test here. 
Will look into it post E2E testing.